docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,235 @@
+``ISMIP6-Projections-Forcing-2300`` for the 2300CE projection protocol.)
+Users also need to manually download the files in the same directory name and
+structure as on the endpoint (including the ``imbie2_basin_numbers_8km.nc``
+file in the ``Ocean_Forcing/imbie2/`` directory, which is provided in the
+globus ``GHub-ISMIP6-Forcing`` endpoint.
+
+In addition to the ISMIP6 forcing files and the basin numbers file, users have
+to download the RACMO modern climatology data `Jan Melchior van Wessem et al
+. (2018) https://tc.copernicus.org/articles/12/1479/2018/#section7>`_.
+if they want to run the ``atmosphere`` test case. The RACMO file is used to
+correct the ismip6 the surface mass balance (SMB) data with the modern
+climatology. Users also have to provide the MALI mesh to which the processed
+forcing files will be saved.
+
+Once the input (source) files are downloaded in the proper directory structure,
+users can configure necessary options (see the below ``config options``
+section for more detail and an example) as well as the ``output_base_path``
+within which the subdirectory called ``atmosphere_forcing/``,
+``basal_melt/`` and ``ocean_thermal_forcing/`` (and further subdirectories that
+match the input file directory structure) are created if the directories do
+not already exist), and processed files will be saved there.
+
+
+config options
+--------------
+
+All three test cases share some set of default config options under the section
+``[ismip6_ais]`` and have separate config options for each test case:
+``[ismip6_ais_atmosphere]``, ``[ismip6_ais_ocean_thermal]``, and
+``[ismip6_ais_ocean_basal]``. In the general config section
+``[ismip6_ais]``, users need to supply base paths to input files and MALI mesh
+file, and MALI mesh name, as well as the model name, climate forcing scenario
+and the projection end year of the ISMIP6 forcing data, which can be chosen
+from the available options as given in the config file (see examples below.)
+In the ``ismip6_ais_atmosphere`` section, users need to indicate
+``True`` or ``False`` on whether to process the RACMO modern climatology
+(``True`` is required to run the ``ProcessSMBRacmo`` step, which needs to take
+precedent to running ``ProcessSMB`` step. In the ``ismip6_ais_ocean_thermal``
+section, users need to indicate ``True`` or ``False`` and on whether to process
+the observational thermal forcing data, which represents the modern ocean
+climatology between 1995-2017. When ``process_obs_data`` is set to ``True``,
+ocean forcing anomaly files from climate models are not processed (these files
+will only be processed when ``process_obs_data`` is set to ``False``)
+
+For all of the config sections specific to each test case, users need to define
+the interpolation scheme among ``bilinear``, ``neareststod`` and ``conserve``
+methods (however, note that the ocean basal test case should always use the
+``neareststod`` method because the source files have a single valued data per
+basin.)
```

Comment:
```suggestion
For most of the config sections specific to each test case, users may choose
the interpolation scheme among ``bilinear``, ``neareststod`` and ``conserve``
methods.  The exception is that the ``ocean_basal`` test case should always use the
``neareststod`` method because the source files have a single valued data per
basin.
```